diff --git a/sources.conf b/sources.conf
index ad4131c..1af174f 100644
--- a/sources.conf
+++ b/sources.conf
@@ -109,80 +109,80 @@ S172E   Eutelsat 172A
 S180E   Intelsat 18
 S177W   NSS 9
 
-# Atlantic
-
-S0.8W   Intelsat 10-02
-S1W     Thor 5/6
-S4W     Amos 2/3
-S5W     Eutelsat 5 West A
-S7W     Nilesat 101/201 & Eutelsat 7 West A
-S8W     Eutelsat 8 West A/C
-S11W    Express AM44
-S12.5W  Eutelsat 12 West A
-S14W    Express A4
-S15W    Telstar 12
-S18W    Intelsat 901
-S20W    NSS 7
-S22W    SES 4
-S24.5W  Intelsat 905
-S27.5W  Intelsat 907
-S30W    Hispasat 1D/1E
-S31.5W  Intelsat 25
-S34.5W  Intelsat 903
-S37.5W  NSS 10 & Telstar 11N
-S40.5W  SES 6
-S43W    Intelsat 11
-S45W    Intelsat 14
-S50W    Intelsat 1R
-S53W    Intelsat 23
-S55.5W  Intelsat 805
-S58W    Intelsat 21
-S61W    Amazonas 2/3
-
 # America
 
-S61.5W  Echostar 16
-S63W    Telstar 14R
-S65W    Star One C1
-S67W    AMC 4
-S70W    Star One C2
-S72W    AMC 6
-S72.7W  Nimiq 5
-S75W    Star One C3
-S77W    QuetzSat 1
-S82W    Nimiq 4
-S83W    AMC 9
-S84W    Brasilsat B4
+S139W   AMC 8
+S137W   AMC 7
+S135W   AMC 10
+S133W   Galaxy 15
+S131W   AMC 11
+S129W   Ciel 2
+S127W   Galaxy 13/Horizons 1
+S125W   Galaxy 14 & AMC 21
+S123W   Galaxy 18
+S121W   Echostar 9/Galaxy 23
+S119W   Echostar 14 & DirecTV 7S
+S118.8W Anik F3
+S116.8W SatMex 8
+S114.9W SatMex 5
+S113W   SatMex 6
+S111.1W Anik F2
+S110W   DirecTV 5 & Echostar 10/11
+S107.3W Anik F1R/G1
+S105W   AMC 15/18
+S103W   AMC 1
+S101W   DirecTV 4S/8 & SES 1
+S99.2W  Galaxy 16
+S97W    Galaxy 19
+S95W    Galaxy 3C
+S93.1W  Galaxy 25
+S91W    Galaxy 17 & Nimiq 6
+S89W    Galaxy 28
+S87W    SES 2
 S85W    AMC 16
 S85.1W  XM 3
-S87W    SES 2
-S89W    Galaxy 28
-S91W    Galaxy 17 & Nimiq 6
-S93.1W  Galaxy 25
-S95W    Galaxy 3C
-S97W    Galaxy 19
-S99.2W  Galaxy 16
-S101W   DirecTV 4S/8 & SES 1
-S103W   AMC 1
-S105W   AMC 15/18
-S107.3W Anik F1R/G1
-S110W   DirecTV 5 & Echostar 10/11
-S111.1W Anik F2
-S113W   SatMex 6
-S114.9W SatMex 5
-S116.8W SatMex 8
-S118.8W Anik F3
-S119W   Echostar 14 & DirecTV 7S
-S121W   Echostar 9/Galaxy 23
-S123W   Galaxy 18
-S125W   Galaxy 14 & AMC 21
-S127W   Galaxy 13/Horizons 1
-S129W   Ciel 2
-S131W   AMC 11
-S133W   Galaxy 15
-S135W   AMC 10
-S137W   AMC 7
-S139W   AMC 8
+S84W    Brasilsat B4
+S83W    AMC 9
+S82W    Nimiq 4
+S77W    QuetzSat 1
+S75W    Star One C3
+S72W    AMC 6
+S72.7W  Nimiq 5
+S70W    Star One C2
+S67W    AMC 4
+S65W    Star One C1
+S63W    Telstar 14R
+S61.5W  Echostar 16
+
+# Atlantic
+
+S61W    Amazonas 2/3
+S58W    Intelsat 21
+S55.5W  Intelsat 805
+S53W    Intelsat 23
+S50W    Intelsat 1R
+S45W    Intelsat 14
+S43W    Intelsat 11
+S40.5W  SES 6
+S37.5W  NSS 10 & Telstar 11N
+S34.5W  Intelsat 903
+S31.5W  Intelsat 25
+S30W    Hispasat 1D/1E
+S27.5W  Intelsat 907
+S24.5W  Intelsat 905
+S22W    SES 4
+S20W    NSS 7
+S18W    Intelsat 901
+S15W    Telstar 12
+S14W    Express A4
+S12.5W  Eutelsat 12 West A
+S11W    Express AM44
+S8W     Eutelsat 8 West A/C
+S7W     Nilesat 101/201 & Eutelsat 7 West A
+S5W     Eutelsat 5 West A
+S4W     Amos 2/3
+S1W     Thor 5/6
+S0.8W   Intelsat 10-02
 
 S360E   Any satellite
 
